config/sites/its.uiowa.edu/config_split.patch.core.entity_view_display.node.page.default.yml

```diff
@@ -0,0 +1,12 @@
+adding:
+  third_party_settings:
+    layout_builder_restrictions:
+      entity_view_mode_restriction:
+        allowlisted_blocks:
+          'Lists (Views)':
+            - 'views_block:alerts_list_block-outages_degradations'
```

Comment:
Am I reading this right? Can we avoid this now that it is tagged with the Site Custom views category?

<img width="739" alt="Screenshot 2024-01-19 at 11 43 49 AM" src="https://github.com/uiowa/uiowa/assets/4663676/00e8b189-6f42-40d1-a7f2-c71e5eac6b74">

Reply:
Messaged Joe personally, he will re-export the block options to fall in line with our desired strategy.

Reply:
locally enabled Devel was also getting added to this.